docs/deployment/rds-s3/README.md

```diff
@@ -167,49 +206,78 @@ Resources:
-5. Configure Kubeflow Pipelines by editing the following file in `awsconfigs/apps/pipeline` directory:
+5. Update the KFP configurations
+    1. [RDS] Configure the `awsconfigs/apps/pipeline/rds/params.env` file with the RDS endpoint url and the metadata db name.
 
-   1. Configure `awsconfigs/apps/pipeline/params.env` file with the RDS endpoint URL, S3 bucket name, and S3 bucket region that were configured when following the steps in Create RDS Instance and Create S3 Bucket steps in prerequisites(#1-prerequisites).
-
-      - For example, if your RDS endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, metadata db name is `kubeflow`, S3 bucket name is `kf-aws-demo-bucket`, and s3 bucket region is `us-west-2` your `params.env` file should look like:
-      - ```
+       For example, if your RDS endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com` and your metadata db name is `kubeflow` your `params.env` file should look like:
+       ```
         dbHost=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
         mlmdDb=kubeflow
-        bucketName=kf-aws-demo-bucket
-        minioServiceHost=s3.amazonaws.com
-        minioServiceRegion=us-west-2
         ```
 
+    2. [S3] Configure the `awsconfigs/apps/pipeline/s3/params.env` file with with the `S3 bucket name`, and `S3 bucket region`..
+
+         For example, if your S3 bucket name is `kf-aws-demo-bucket` and s3 bucket region is `us-west-2` your `params.env` file should look like:
+         ```
+          bucketName=kf-aws-demo-bucket
+          minioServiceHost=s3.amazonaws.com
+          minioServiceRegion=us-west-2
+          ```
+
 ## 3.0 Build Manifests and Install Kubeflow
+
 Once you have the resources ready, you can continue on to deploying the Kubeflow manifests using the single line command below -
 
+Choose your deployment option below from:
+
+- Deploying the configuration for both RDS and S3
+- Deploying the configuration for RDS only
+- Deploying the configuration for S3 only
+
+#### [RDS and S3] Deploy both RDS and S3
+
 ```sh
 while ! kustomize build docs/deployment/rds-s3 | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
 ```
 
+#### [RDS] Deploy RDS only
+
+```sh
+while ! kustomize build docs/deployment/rds-s3/rds-only | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+```
+
+#### [S3] Deploy S3 only
+
+```sh
+while ! kustomize build docs/deployment/rds-s3/s3-only | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+```
+
 Once, everything is installed successfully, you can access the Kubeflow Central Dashboard [by logging in to your cluster](../vanilla/README.md#connect-to-your-kubeflow-cluster).
 
 Congratulations! You can now start experimenting and running your end-to-end ML workflows with Kubeflow.
 
-## 4.0 Verify the installation
+## 4.0 Using only RDS or only S3
 
-### 4.1 Verify RDS
+### RDS only
+Follow the steps prefixed with [RDS] above.
```

Comment:
links missing in `[]`?

Also what is this section trying to do?

Reply:
`[RDS]` is intended to be a label for a section instead of a link

I was trying to explain what the label meant in terms of performing a decoupled installation

Reply:
I have moved this section to the beginning of the installation instructions.